docs/source/1.0/spec/core/idl.rst

```diff
@@ -1031,10 +1029,9 @@ The following example defines a union shape with several members:
         union MyUnion {
             i32: Integer,
 
-            stringA: String,
+            string: String,
 
-            @sensitive
```

Comment:
This was to show that traits can also be applied to members of unions, can one be added here?

Reply:
oh makes sense that's why there were 2 strings. added a length trait to the string.